code/game/objects/machinery/computer/intel_computer.dm

```diff
@@ -55,6 +55,11 @@
 		SSpoints.dropship_points += dropship_reward
 		minor_announce("Classified transmission recieved from [get_area(src)]. Bonus delivered as [supply_reward] supply points and [dropship_reward] dropship points.", title = "TGMC Intel Division")
 		SSminimaps.remove_marker(src)
+		return
+	if (machine_stat & NOPOWER) /// checks for lack of power and shuts down
+		printing = FALSE
+		progress = 0
+		visible_message("<b>[src]</b> shuts down as it loses power. Any data is lost.")
```

Comment:
I think this is suboptimal? I need more input.

At the moment the IC doesn't even check for power, if power is cut it'll visually be disabled, but keep progressing and eventually submit for points. I tried to copy the nuke disks implementation but instead fell back to the computer.dm
`/obj/machinery/computer/process()`
    `if(machine_stat & (NOPOWER|BROKEN|DISABLED))`